### PR TITLE
Update Web3Auth to include whitelabel and latest SDK support

### PR DESCRIPTION
**Pull request by shahbaz17**
Hi team,

I am one of the team members from @web3auth and wanted to update the SDK to include the latest changes.

Please take a look.

Thanks.

### CODE REVIEW

**Comment by arein** [comment]
Can you please provide a better description + screenshots

**Comment by chrischip** [comment]
this is exactly what I need

File: src/providers/connectors/web3auth.ts [hunk 1]
```diff
@@ -16,16 +19,23 @@ const connectToweb3auth = async(Web3Auth: any, opts: IWeb3AuthConnectorOptions)
             const options = opts || {};
             const chainID = options.chainId || "0x1"
             const clientID = options.clientId || "localhostid"
-            const infuraID = options.infuraId
+            const rpcTarget = options.rpcTarget || "https://rpc.ankr.com/eth"
+            const theme = options.theme || "light"
+            const loginMethodsOrder = options.loginMethodsOrder || []
+            const appLogo = options.loginMethodsOrder || ""
```
Comment (chrischip) [inline src/providers/connectors/web3auth.ts:25]: this line is incorrect....

**Review by chrischip — CHANGES_REQUESTED** [review]
appLogo not correctly obtained from options

**Comment by shahbaz17** [comment]
Hey team,

We recently renamed our `@web3auth/web3auth` SDK to `@web3auth/modal` and added more support. Please check out [here](https://web3auth.io/docs/sdk/web/modal/).

I'll be updating this PR to include the latest changes from https://web3auth.io/docs/sdk/web/modal/

**Comment by 0xAsimetriq** [comment]
With stable version 2.0.0 of Web3Modal now released, we are officially dropping support for version 1.x
Due to this this issue/pr was marked for closing. It is highly recommended to upgrade as 2.x will be receiving further updates that will enable functionality for some of our newer sdks like auth and push as well as support for WalletConnect v2 (See this post about WalletConnect v1 being deprecated https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e)

If you need to continue using Web3Modal 1.x and require this feature/fix implemented, we suggest adding it via forking V1 branch.

**Pull request closed without merging**